projects/demo-playwright/tests/kit/input-month/input-month.pw.spec.ts

```diff
@@ -99,20 +90,6 @@ describe('InputMonth', () => {
                 await expect(inputMonth.calendar).toBeAttached();
             });
 
-            test('does NOT open dropdown on click on calendar icon for disabled input', async ({
-                page,
-            }) => {
-                await tuiGoto(
-                    page,
-                    `${DemoRoute.InputMonth}/API?disabled=true&readOnly=false`,
-                );
-
-                await expect(inputMonth.calendar).not.toBeAttached();
-
-                await inputMonth.clickOnIcon();
```

Comment:
Let improve `TuiInputMonthPO.clickOnIcon` inside `projects/demo-playwright/utils/page-objects/input-month.po.ts`:

```ts
public async clickOnIcon(
    options: Parameters<Locator['click']>[0] = {},
): Promise<void> {
    // [...]

    await this.host.click({
        ...options, // <-----------
        position: {x: box.width - padding - iconWidth, y: box.height / 2},
    });
}
```

Then
```suggestion
                await inputMonth.clickOnIcon({force: true});
```